### PR TITLE
Feature/0000001 add basic cli

### PR DESCRIPTION
**Pull request by theinfamousrj**
Attempting to parse YAML file with CLI. Lots of hardcoded things I think we can easily rip and replace with what Edvin is working on.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+	"strings"
+
+	"github.com/smallfish/simpleyaml"
+)
+
+/*
+	I am sure there's a better way to do this, I'm just trying to get the hang of Go right now.
+	Edvin probably has something better for CLI stuff already.
+*/
+func main() {
```
Comment (bricerising) [inline main.go:16]: Hmm I'm ok with merging this as a first pass at a cli, but think we should use one of these two projects for the cli we go with in the end

https://github.com/spf13/cobra
https://github.com/urfave/cli

**Review by bricerising — APPROVED** [review]

File: main.go (hunk 1 above)
Comment (dizveloper) [reply main.go:16]: I'm messing around with cobra on my branch right now.

Comment (theinfamousrj) [reply main.go:16]: Edvin got the goodies. It was more of a learning experience for me with Go. I actually kinda like Go now and I think Edvin might be able to use my YAML parser to some extent. We can and will make it better. 😺

Comment (dizveloper) [reply main.go:16]: I did indeed use the yaml parser ^ @theinfamousrj

**Pull request merged**